students/fill0s0fear/hw-1/second.js

```diff
@@ -0,0 +1,3 @@
+let  str = prompt('Enter a text');
```

Comment:
You already have `str` variable in the code

Reply:
I use a code from my past course and second script was supposed to be the second task which unrelated with the first script.
For the first homework I just write those 2 scripts in code one by one

Reply:
hm, but you still have an error...

Reply:
changed in new commit (3rd)